sound/soc/sof/topology.c

```diff
@@ -359,6 +359,14 @@ int get_token_uuid(void *elem, void *object, u32 offset)
 	return 0;
 }
 
+int get_token_string(void *elem, void *object, u32 offset)
+{
+	u8 *dst = (u8 *)object + offset;
+
+	memcpy(dst, elem, SNDRV_CTL_ELEM_ID_NAME_MAXLEN);
+	return 0;
+};
+
```

Comment:
The commit message is not clear enough on the problem statement, why you need this and how it will be used.